### DISCUSSION TITLE
Sort by "Date Added to Collection"

### DISCUSSION THREAD

**Discussion by goodtrailer** [ppy/osu#29606] [Ideas]
If a collection is selected, then have a sorting option to sort by Date Added to Collection (I think this is pretty self explanatory?). If no collection is selected, the option could be grayed out, or it could work the same as Date Added.

Reasoning: There are plenty of use cases IMO, but here's one from me. I have a lot of maps that I downloaded a while ago but only got around to playing now, so when I add it to a large-ish collection, it's very difficult to find if I don't remember the name. Sorting by Date Added doesn't help here because it was downloaded a while ago, so it gets buried somewhere in the middle. So I think sorting by Date Added to Collection would be nice.

**Comment by Lyti4n** [comment]
I think this would be a very good addition, in particular I have a lot of issues with the way date added works right now with individual diffs but I don't think the solution I would prefer would please everyone, so having a way to sorta do it manually would fix that.
